src/main/webapp/go.js

```diff
@@ -33,10 +33,12 @@ const FINAL_MSSG = 'Congrats, you\'ve finished the hunt!';
 // Other constants.
 const INDEX_PARAM = 'new-index';
 const INVISIBLE_CLASS = 'invisible';
+const REFRESH_TIME = 10000; //ten seconds
 
 // Global variables.
 const puzzleArr = [];
 let destIndex;
+let hintIndex = 0; // Marks the hint that the user will see next.
```

Comment:
Should this be reset to 0 when the proceed to the next destination?

Reply:
The page reloads once the user submits an answer, so hintIndex is automatically set to 0 when the user begins to find the next destination.